### PR TITLE
fix(jazzy-desktop): remove execstack from libamdhip64.5

### PR DESCRIPTION
**Pull request by Guillaumebeuzeboc**
Solves https://github.com/ubuntu-robotics/ros-content-sharing-snaps/issues/15 for the jazzy-desktop content sharing snaps.

### CODE REVIEW

**Comment by Guillaumebeuzeboc** [comment]
The generated snap is passing the `review-tools.snap-review`

**Review by MirkoFerrati — APPROVED** [review]
lgtm!

File: templates/snapcraft.yaml.j2 [hunk 1]
```diff
@@ -170,3 +170,16 @@ parts:
     override-prime: |
       rm -vf usr/lib/jvm/java-11-openjdk-*/lib/security/blacklisted.certs
 {% endif %}
+{% if ros_distro == 'jazzy' and variant == 'desktop' %}
+  humble-desktop-libamdhip-exec-stack:
```
Comment (artivis) [inline templates/snapcraft.yaml.j2:174]: @Guillaumebeuzeboc @MirkoFerrati This part is prefixed `humble` when it concerns `jazzy`...

Comment (MirkoFerrati) [reply templates/snapcraft.yaml.j2:174]: I see, we should remove humble from the prefix as it's just a name and it should refer to the library we are fixing, not the ros distro we are using

**Pull request merged**